README.md

```diff
@@ -1,6 +1,49 @@
-# Circles Dashboard
+<div align="center">
+	<img width="80" src="https://raw.githubusercontent.com/CirclesUBI/.github/main/assets/logo.svg" />
+</div>
 
-Status dashboard and trust network explorer for [Circles](https://github.com/CirclesUBI).
+<h1 align="center">circles-dashboard</h1>
+
+<div align="center">
+ <strong>
+   Status dashboard and trust network explorer for [Circles]
```

Comment:
The Circles link has to be defined somewhere @louilinn :-)

Reply:
I actually think that link is not necessary. In the rest of the repos we don't include a link here.

Reply:
Or, we could add a link to the dashboard in prod?
https://dashboard.circles.garden/